Content.Client/UserInterface/OptionsMenu.Graphics.cs

```diff
@@ -97,6 +101,22 @@ public GraphicsControl(IConfigurationManager cfg)
                     }
                 });
 
+                _hudThemeOption = new OptionButton();
+                _hudThemeOption.AddItem(Loc.GetString("ui-options-hud-theme-default"));
+                _hudThemeOption.AddItem(Loc.GetString("ui-options-hud-theme-modernized"));
+                _hudThemeOption.AddItem(Loc.GetString("ui-options-hud-theme-classic"));
```

Comment:
instead of hardcoding the theme options, enumerate the HUD theme prototypes using `IPrototypeManager.EnumeratePrototypes<T>`